proposal-template.md

```diff
@@ -1,16 +1,36 @@
-# ECMAScript proposal: @Name
+# ECMAScript proposal: Package
 - [Motivation](#motivation)
 - [High-level API](#high-level-api)
 - [FAQ](#faq)
 
 ## Motivation
 
-Proposal motivation
+С багами npm и yarn уже невозможно жить, и вообще это лишние звено.
+Все используют какие-то библиотеке, в такой ситуации логично что язык должен взять на себя это бремя. Было круто указывать ссылку на гитхаб с библиотекой и использовать ее сразу. А не как сейчас перейти в консоль, запустить npm/yarn, ждать, пересобирать и только сейчас использовать!
+
+Это нововведение значительно ускорит разработку!
 
```

Comment:
It's impossible to live with the bugs in npm and yarn anymore, and in general it's an extra step. Everyone is using some libraries, in this situation it's reasonable that the language should take it upon itself. It'd be awesome to use a link to a library on github and immediately be able to use it. Instead of the current: going to the console, running npm/yarn, waiting, rebuilding, and only then using it.

This new thing will speed up development significantly